vmd-server/src/server/cli.rs

```diff
@@ -0,0 +1,70 @@
+// === Cli ====================================================================
+//
+// Command line interface for the virtual machine daemon server.
+//
+// === External crates ========================================================
+
+use std::{
+    path::PathBuf,
+    fmt::{Display, Formatter, Result},
+};
+use clap::Parser;
+
+// === Internal modules =======================================================
+
+use vmd_rust_server_api::{
+    BASE_PATH,
+    API_VERSION,
+};
+
+// === Implementations ========================================================
+
+/// A virtual machine daemon server
+#[derive(Parser, Debug)]
+pub(crate) struct Args {
+
+    /// Server hostname
```

Comment:
These comments are also self-evident. No need imo.

Reply:
These comments are actually gonna be rendered in the cli when using `--help`.

```
❯ cargo run -p vmd_server --release -- --help
    Finished release [optimized] target(s) in 0.25s
     Running `target/release/vmd_server --help`
A virtual machine daemon server

Usage: vmd_server [OPTIONS] --hostname <HOSTNAME> --port <PORT> --cacert <CACERT> --cert <CERT> --key <KEY>

Options:
  -h, --hostname <HOSTNAME>  Server hostname
  -p, --port <PORT>          Port to listen on
  -a, --cacert <CACERT>      Path to certificate authority pem file (.pem)
  -c, --cert <CERT>          Path to server certificate pem file (.pem)
  -k, --key <KEY>            Path to server private key pem file (.pem)
      --ocsp <OCSP>          Unimplemented!
  -h, --help                 Print help
  ```

Reply:
Great reason. I did not realize that.